src/Symfony/Component/Serializer/Tests/Normalizer/ObjectNormalizerTest.php

```diff
@@ -31,10 +30,8 @@
 use Symfony\Component\Serializer\SerializerInterface;
 use Symfony\Component\Serializer\Tests\Fixtures\CircularReferenceDummy;
 use Symfony\Component\Serializer\Tests\Fixtures\GroupDummy;
-use Symfony\Component\Serializer\Tests\Fixtures\MaxDepthDummy;
```

Comment:
should the fixture file be removed? if yes what about other that may now be unused?

Reply:
Well, this one is still used by many tests (see `\Symfony\Component\Serializer\Tests\Normalizer\Features\MaxDepthTestTrait::testMaxDepthHandler`) for instance.
But I'll have a closer look at it soon :)